modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/SqlQueryProcessor.java

```diff
@@ -287,7 +289,11 @@ public synchronized CompletableFuture<Void> startAsync(ComponentContext componen
         var storageProfileValidator = new ClusterWideStorageProfileValidator(logicalTopologyService);
         var nodeFilterValidator = new ClusterWideNodeFilterValidator(logicalTopologyService);
 
-        var ddlSqlToCommandConverter = new DdlSqlToCommandConverter(storageProfileValidator, nodeFilterValidator);
+        Supplier<TableStatsStalenessConfiguration> stalenessProperties = () -> new TableStatsStalenessConfiguration(
+                clusterCfg.createTable().staleRowsFraction().value(),
+                clusterCfg.createTable().minStaleRowsCount().value());
```

Comment:
two invocation of `value()` returns two different views of configuration. In other words, this may result in properties combination which never exists (i.e. value of `staleRowsFraction` from one state and value of `minStaleRowsCount ` from another). It's better to get view from `createTable()` and then read necessary properties